schema/deploy/warehouse/encounter/triggers/update-modified-timestamp.sql

```diff
@@ -0,0 +1,12 @@
+-- Deploy seattleflu/schema:warehouse/encounter/triggers/update-modified-timestamp to pg
+-- requires: warehouse/encounter
+-- requires: warehouse/encounter/triggers/update-modified-timestamp
```

Comment:
Looks like `warehouse/encounter/triggers/update-modified-timestamp` was used by mistake instead of `functions/update-modified-timestamp` both at the top of deploy files and in the plan file.

Reply:
Yep, sure enough. I fixed it. 

The pains of naming things too similarly....